client.go

```diff
@@ -59,7 +59,7 @@ func NewClient(locale, username, password string) Client {
 		username: username,
 		password: password,
 		live:     true,
-		timeout:  0,
+		client:   http.DefaultClient,
```

Comment:
as `net/http.DefaultClient` is just:

```
$> go doc net/http.DefaultClient 
package http // import "net/http"

var DefaultClient = &Client{}
    DefaultClient is the default Client and is used by Get, Head, and Post.

```

I think it would just be as well to write:
```suggestion
		client:   &http.Client{},
```

Reply:
using `&http.Client{}` instead `http.DefaultClient` won't change the functionality of the `Timeout(time.Duration)` function, is it better to use the pointer to the empty struct?

Reply:
sorry, I am not sure I understand what you mean.

Reply:
yeah sorry, I mean `DefaultClient` it's the same as the empty struct (`&http.Client{}`). So my question is: Is it better to use `&http.Client{}`? (really just a knowledge and best-practice question)

Reply:
operationally, it's the same (from `openfoodfacts.Client` 's point of view, it's the same), but our client's internals would rely on the state of a global variable, so we might end up with some weird "action at a distance" types of bugs if someone were to modify the state of `net/http.DefaultClient` under our feet.
(and we might be the source of such bugs if we were using that global variable and modify it via our `Timeout` method.)

best is really to use our own `*net/http.Client` value :)

Reply:
true, thx!, gonna update rn